src/llfs/data_packer.hpp

```diff
@@ -138,9 +138,9 @@ class DataPacker
   }
 
   template <typename T>
-  [[nodiscard]] T* pack_record(const batt::StaticType<T>& = {})
+  [[nodiscard]] T* pack_record(const batt::StaticType<T>& = {}, const usize count = 1)
```

Comment:
Nice optimization!  Would you please add a doc comment for this function that includes a description of the new parameter?